apps/server/src/adapters/index.test.ts

```diff
@@ -1,104 +1,97 @@
-import { http, HttpResponse } from 'msw'
-import { ConnectionStatus, OAuthStatus } from '../shared/contract'
-import { READ_MEMBER_STATUS_PATH } from '../test/handlers'
-import { server } from '../test/testServer'
-import { AggregatorAdapterBase } from './index'
-import { MxAdapter } from './mx'
+import { ConnectionStatus, OAuthStatus } from "../shared/contract";
+import { AggregatorAdapterBase } from "./index";
+import { TestAdapter } from "../test-adapter/adapter";
 
-const testConnectionId = 'test_connection_id'
+const testAdapter = new TestAdapter({
+  labelText: "testLabelText",
+  aggregator: "testAdapterA",
+});
 
-const isIntEnv = false
-const mxAdapter = new MxAdapter(isIntEnv)
+const testConnectionId = "test_connection_id";
 
 const aggregatorAdapterBase = new AggregatorAdapterBase({
   context: {
-    aggregator: 'mx',
+    aggregator: "testAdapterA",
     auth: {
-      token: 'test_token',
-      iv: 'test_iv'
+      token: "test_token",
+      iv: "test_iv",
     },
-    resolved_user_id: 'test_user_id'
-  }
-})
+    resolved_user_id: "test_user_id",
+  },
+});
 
-describe('AggregatorAdapterBase', () => {
-  describe('getOauthState', () => {
+describe("AggregatorAdapterBase", () => {
+  describe("getOauthState", () => {
     beforeAll(async () => {
-      await aggregatorAdapterBase.init()
-      aggregatorAdapterBase.aggregatorAdapter = mxAdapter
-    })
+      await aggregatorAdapterBase.init();
+      aggregatorAdapterBase.aggregatorAdapter = testAdapter;
```

Comment:
Why don't you pass in a context that assigns this adapter?

Reply:
https://github.com/Universal-Connect-Project/ucw-app/blob/b46ce0ee016998a9c7b11c599b5612f7b119fdb3/apps/server/src/adapters/index.ts#L65